design/crud-api/db/repository/postgres/postgres_client.go

```diff
@@ -60,3 +62,127 @@ func (c *Client) Close() error {
 func (c *Client) DB() *sql.DB {
 	return c.db
 }
+
+// InitializeTables creates the necessary tables if they don't exist
+func (c *Client) InitializeTables(ctx context.Context) error {
+	// Create entity_attributes table
+	entityAttributesSQL := `
+	CREATE TABLE IF NOT EXISTS entity_attributes (
```

Comment:
@Isuru-rangana shall we explain a bit why we are creating the `entity_attributes` table? or should this have the name of the key we get from the map of attributes?

Reply:
Let's add a comment for this in the code. Let's be as descriptive as possible. Also explain the purpose of this table clearly and how it is used in the code.